docs/tuning_fledge.rst

```diff
@@ -762,6 +767,113 @@ When collection is enabled the following counters will be collected for the stor
       - The size of the JSON payload in the delete calls to the storage layer for the given table.
       - There is little an end user can influence regarding the payload size, however it gives an indication of bandwidth usage for the storage API.
 
+Purge
+=====
+
+The purpose of the purge processes within Fledge is to control the usage of the storage system. Fledge has two different purge process that run, each of these purges a different aspect of the storage within the system;
+
+  - **System Purge** - The system purge process is responsible for purging the logs held internally within the Fledge storage system. There are three types of log information held in the storage system; statistics, the audit trail and task execution history.
+
+    .. note::
+
+        The *System Logs*, or message logs, are not held within the Fledge storage system but are rather sent to the Linux system logging facility, *syslog*. This is configured within the Linux system itself to rotate, compress and ultimately remove logs using the system define log rotation settings.
+
+  - **Purge** - The purge process is responsible for purging the readings data from the system. 
+
+Purge System
+------------
+
+The log purging is perhaps the simpler of the two purge process to discuss as it has the least impact on the performance of the system. The configuration of the process itself can be found under the *Configuration* menu option in the *Utilities::Purge System* category.
+
++---------------------+
+| |PurgeSystemConfig| |
++---------------------+
+
+The configuration options merely allow you to set the number of days worth of data that should be retained for each of the three log categories. The important consideration here is that the various logs should not be allowed to grow to such an extent that you risk exhausting the storage system, but that should should retain sufficient information to be able to examine enough history of the system.
+
+The other dimension to consider is that performance is known to degrade as these tables become large, it is therefore not simply keeping an extensive history just because you have the storage to do so. Reducing the history kept can improve the performance.
+
+Typically the statistics that are held will take the most space in the system, especially if you are collecting per asset ingest statistics and you collect data for many assets.  There are actually two forms of statistics kept, the absolute counters and the history snapshot of the statistics. The history snapshot records the statistics values every 15 seconds and create an entry in the statistics history table for each statistic every 15 seconds. It is these statistic history entries that are purged and not the absolute statistics counters. Hence the retention period for statistics, the statistics history, is generally lower.
```

Comment:
"_There are actually two forms of statistics kept_" Follow this with a colon instead of a comma.